wordpress-mobile/WordPress-Android
# Updates: the tab bar icons and styling

Fixes #

## Description
### Updates the icons in bottom bar nav in JP/WP app


| Dark  | Light Mode | 
|---|---|
| <img width="843" alt="Screenshot 2024-01-03 at 3 51 14 PM" src="https://github.com/wordpress-mobile/WordPress-Android/assets/17463767/267a1ecf-db25-4b93-a58d-e0e61c7817fb">  | <img width="788" alt="Screenshot 2024-01-03 at 3 51 00 PM" src="https://github.com/wordpress-mobile/WordPress-Android/assets/17463767/517a8605-f6a8-4e21-a9a8-493364c416fe">  |   


### Sets the me icon to B&W or with color depending on the selected state 
[me_icon_tint.webm](https://github.com/wordpress-mobile/WordPress-Android/assets/17463767/a24b6e0e-7a8f-47dc-976f-8b44a08aa647)


### Adds a pop animation to the bottom nav icon on selecting 

Note: please ignore the badge position on notification icon. It has been moved to left of the icon as before

[Screen_recording_20240103_232835.webm](https://github.com/wordpress-mobile/WordPress-Android/assets/17463767/3a022c9d-485e-4f35-a277-c473b846fb30)

## To Test:
- Login to the JP/WP app 
- Go to the dashboard 
- Verify that the icons on the bottom bar as in the design P4DK0f5zK8F3Y1eifGqS39-fi-3:12381
- Verify new tab icons adapt to environment (light vs dark)
- Verify the icons animate when switching tabs
- Verify the me icon gets blacked out when not selected
- Verify notification icon shows the correct badge color when there are unread notifications (blue for WP, green for JP)

## Regression Notes

1. Potential unintended areas of impact
Bottom bar icons are not correct

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing 

3. What automated tests I added (or what prevented me from doing so)
N/A
  
PR submission checklist:

UI Changes testing checklist:
